app/features/Type/Notes.py

```diff
@@ -1,13 +1,16 @@
 class Notes:
     def __init__(self):
-        self.notesWithRepetition = []  # {'note': , 'repetition': , 'time': }
+        self.notesWithRepetition = []  # {'note': , 'repetition': , 'percentage': }
```

Comment:
I think the object may look more like this:
```
{'note': , 'percentage':  }
```
I'm not sure we need the repetitions.

Reply:
As you want, as long as I have the info, I figured it could only be a plus to display it in addition to the percentage (even if it is not used afterwards).